p4proto/automake.mk

```diff
@@ -20,6 +20,8 @@ EXTRA_DIST += p4proto/kctrl/switchlink/README.md
 EXTRA_DIST += p4proto/kctrl/switchsai/README.md
 EXTRA_DIST += p4proto/p4src/linux_networking/README.md
 EXTRA_DIST += p4proto/p4src/linux_networking/topology.PNG
```

Comment:
We shouldn't add CT's p4 file under this linux_networking directory.
If CT's p4 file need to be added it will be in a different parent directory and it should have its own readme.md file and the topology.png.

Reply:
I see [lnw_ct.p4](https://github.com/ipdk-io/ovs/blob/ovs-with-p4/p4proto/p4src/linux_networking/lnw_ct.p4) at this location. Should this be removed then?

Reply:
Tables from lnw_ct.p4 will be called as part of existing linux_networking.p4 apply block.
So there is no specific testing for this.
This needs to be validated along with existing linux_networking and linux_networking with ECMP.